client/src/app/app.component.html

```diff
@@ -15,47 +15,16 @@
               <app-employee></app-employee>
             </ng-template>
           </li>
-          <li [ngbNavItem]="2">
-            <a ngbNavLink>Payroll</a>
+          <li [ngbNavItem]="'payrolls'">
```

Comment:
Here's a good place to use the attribute binding instead of the property binding so you do need to use nested quotes:

`<li ngbNavItem="payrolls">`